doc/funcref.sgml

```diff
@@ -3450,6 +3458,72 @@ returns one of the constants<itemize>
 </quote>
 
 
+<sect1>getcurrentdevice<label id="getcurrentdevice"><p>
+
+<quote>
+<descrip>
+<tag/Function/Get current device.
+<tag/Header/<tt/<ref id="device.h" name="device.h">/
+<tag/Declaration/<tt/unsigned char getcurrentdevice (void);/
+<tag/Description/The function returns the current device.
+<tag/Notes/<itemize>
+<item>
+</itemize>
+<tag/Availability/cc65
+<tag/See also/
+<ref id="getdevicedir" name="getdevicedir">,
+<ref id="getfirstdevice" name="getfirstdevice">,
+<ref id="getnextdevice" name="getnextdevice">
+<tag/Example/<verb>
+puts (getdevicedir (getcurrentdevice (), buf, sizeof buf));
```

Comment:
This isn't a great example as the result is (most of the time) identical to getcwd().
Better `dio_open (getcurrentdevice ());`.